sdk/include/opentelemetry/sdk/trace/recordable.h

```diff
@@ -21,6 +23,16 @@ class Recordable
 public:
   virtual ~Recordable() = default;
 
+  /**
```

Comment:
In other languages `Recordable` is not an interface but a concrete immutable class.